hadoop-ozone/dist/src/main/compose/upgrade/testlib.sh

```diff
@@ -100,13 +100,13 @@ run_test() {
 ## @description Generates data on the cluster.
 ## @param The prefix to use for data generated.
 generate() {
-    execute_robot_test scm -v PREFIX:"$1" upgrade/generate.robot
+    execute_robot_test scm -v PREFIX:"$1" ${@:2} upgrade/generate.robot
```

Comment:
nit. Can we update the comments for these functions to indicate that all arguments after the first are passed directly to robot?

Reply:
Done